lldb/source/Plugins/Process/Utility/LinuxSignals.cpp

```diff
@@ -27,6 +27,30 @@
 #ifndef SEGV_CPERR
 #define SEGV_CPERR 10
 #endif
+#ifndef SI_QUEUE
+#define SI_QUEUE -1
+#endif
+#ifndef SI_TIMER
+#define SI_TIMER -2
+#endif
+#ifndef SI_MESGQ
+#define SI_MESGQ -3
+#endif
+#ifndef SI_ASYNCIO
+#define SI_ASYNCIO -4
+#endif
+#ifndef SI_SIGIO
+#define SI_SIGIO -5
+#endif
+#ifndef SI_TKILL
+#define SI_TKILL -6
+#endif
```

Comment:
That's overdoing it a bit I'd say. I think we can remove these, as they are really, really old. We can easily add them back if any one turns out to be missing somewhere, but I doubt that will happen.

Reply:
Originally I just did DE_THREAD, but because they were all moved to ASM generic in the same commit in the Linux Kernel I thought it would be safest to do all of them